src/databasetasks.h

```diff
@@ -47,6 +47,7 @@ class DatabaseTasks : public ThreadHolder<DatabaseTasks>
 		void threadMain();
 	private:
 		void runTask(const DatabaseTask& task);
+		void flushInternal();
```

Comment:
I'd prefer if this function would just be on the cpp. You don't need to expose it.